sdk/template/Azure.Template/tests/MiniSecretClientLiveTests.cs

```diff
@@ -8,18 +8,26 @@
 
 namespace Azure.Template.Tests
 {
+    [ClientTestFixture(
+        // List all supported service versions below.
+        MiniSecretClientOptions.ServiceVersion.V7_0
+    )]
     public class MiniSecretClientLiveTests: RecordedTestBase<MiniSecretClientTestEnvironment>
     {
-        public MiniSecretClientLiveTests(bool isAsync) : base(isAsync)
+        private readonly MiniSecretClientOptions.ServiceVersion _serviceVersion;
+
+        public MiniSecretClientLiveTests(bool isAsync, MiniSecretClientOptions.ServiceVersion serviceVersion)
+            : base(isAsync, null /* RecordedTestMode.Record /* delete "null /* " to re-record */)
```

Comment:
If we stick with using the parameter can we come up with an easier way to switch it in code? While the comment juggling is cool I had to read the comment a few times to understand how to do it.

Reply:
Sure. All Ted's comments always just say `null /* RecordedTestMode.Record /* to re-record */` but I thought this might be more descriptive. But, you're right, reading it again later, it makes less sense. How do you like the original, or do you have another suggestion?

I was also thinking about adding a line comment above about the *eng/nunit.config* file if people wanted to force all their tests to live, force-record, etc. How does that sound?